src/storage/hummock_sdk/src/key.rs

```diff
@@ -547,12 +555,18 @@ impl UserKey<Vec<u8>> {
 /// [`FullKey`] is an internal concept in storage. It associates [`UserKey`] with an epoch.
 ///
 /// The encoded format is | `user_key` | `epoch` |.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Default)]
+#[derive(Clone, Copy, PartialEq, Eq, Hash, Default)]
 pub struct FullKey<T: AsRef<[u8]>> {
     pub user_key: UserKey<T>,
     pub epoch: HummockEpoch,
 }
 
+impl<T: AsRef<[u8]>> Debug for FullKey<T> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "FullKey {{ {:?}, {} }}", self.user_key, self.epoch)
```

Comment:
Just a question: Do we have a better way to print &[u8] (in a more human-friendly way)?

Reply:
I implemented the Debug trait for all the key structus to [print hex string](https://github.com/risingwavelabs/risingwave/pull/8580/files#diff-e8546b48ec0e07c8f2c4d526da0e83dc956d44ec12929864691d9b82ed06418aR375) in this PR. This is more human-friendly than printing out a u8 array.